src/mcp_server_helm/server.py

```
@@ -1,1682 +1,27 @@
-        formatted_output = "REPOSITORY LIST:\n\n"
-        formatted_output += "NAME\t\tURL\n"
-
-        for repo in repos:
-            formatted_output += f"{repo.get('name', 'N/A')}\t\t"
-            formatted_output += f"{repo.get('url', 'N/A')}\n"
-
-        return formatted_output
-    except json.JSONDecodeError:
-        # If output is not JSON, return raw output
-        return f"Repository list:\n{output}"
+from .core.server import HelmMCPServer
```

Pylint (3.3.7)

[convention] Import "from .core.server import HelmMCPServer" should be placed at the top of the module

(C0413)